pymathics/graph/base.py

```diff
@@ -86,7 +86,7 @@ def graph_helper(
     root: Optional[int] = None,
     *args,
     **kwargs,
-) -> Optional[Callable]:
+) -> Optional["Graph"]:
```

Comment:
@rocky, if you are in this, maybe a docstring here would be useful too.

Reply:
I do not have a details idea of what graph_heper does and this will probably change. That said, I'll add something minimal and then we can fill it out later.

Reply:
Thanks! Makes sense.